OpenOversight/app/main/views.py

```diff
@@ -30,12 +48,165 @@ def get_officer():
     return render_template('input_find_officer.html', form=form)
 
 
-@main.route('/label', methods=['GET', 'POST'])
-def label_data():
+@main.route('/tagger_find', methods=['GET', 'POST'])
+def get_ooid():
     form = FindOfficerIDForm()
     if form.validate_on_submit():
         return redirect(url_for('main.get_tagger_gallery'), code=307)
-    return render_template('label_data.html', form=form)
+    return render_template('input_find_ooid.html', form=form)
+
+
+@main.route('/label', methods=['GET', 'POST'])
+def get_started_labeling():
+    return render_template('label_data.html')
+
+
+@main.route('/sort', methods=['GET', 'POST'])
+@login_required
+def sort_images():
+    # Select a random unsorted image from the database
+    image = Image.query.filter_by(contains_cops=None).first()
+    if image:
+        proper_path = serve_image(image.filepath)
+    else:
+        proper_path = None
+    return render_template('sort.html', image=image, path=proper_path)
+
+
+@main.route('/tutorial')
+def get_tutorial():
+    return render_template('tutorial.html')
+
+
+@main.route('/user/<username>')
```

Comment:
I figured people might want to show their activity to people that don't have an active OpenOversight account (might help with recruiting), so I left this one as not requiring login. One can't easily get a list of all accounts (could enforce this by modifying the URL here but for the cop threat model I don't think we need to worry about it 😉  - but am happy to revisit if people get concerned about retaliation (e.g. by making the URLs unpredictable or by restricting to login)).